Firebase/android/app/src/main/res/layout/merge_user_item_view.xml

```diff
@@ -2,21 +2,15 @@
 <merge xmlns:android="http://schemas.android.com/apk/res/android">
 
   <LinearLayout
+    style="@style/User.Item"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:orientation="horizontal"
-    android:gravity="center_vertical"
```

Comment:
I guess by the argument of keeping orientation in the layout because it affects children layout, gravity on viewgroups should also stay in the layout. we don't do it in other projects but maybe we should